fast64_internal/sm64/sm64_geolayout_classes.py

```diff
@@ -380,9 +380,9 @@ def to_binary(self, segmentData):
 
     def to_c(self, depth):
         if self.node is not None:
-            nodeC = self.node.to_c()
+            nodeC = self.node.to_c(depth)
```

Comment:
why do this, it seems to me that the `depth` argument is never used by any "node" type object

Reply:
just for consistency